src/smart-components/group/role/group-roles.js

```diff
@@ -58,6 +58,7 @@ const addRoleButton = (isDisabled) => (isDisabled
     </div>
   </Tooltip>
   : <Button
+    id="add-role-btn"
```

Comment:
Is adding ID really necessary? I'd like to avoid as much as possible, if you want to find element in shadow dom while testing you can use `at` method.